fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidConstants.java

```diff
@@ -54,6 +57,20 @@ public static long fromBucketFraction(long numerator, long denominator) {
 		}
 	}
 
+	// ==========================
+	// ==== FLUID ATTRIBUTES ====
+	// ==========================
+	public static final int WATER_TEMPERATURE = 300;
+	public static final int LAVA_TEMPERATURE = 1300;
+
+	public static final int WATER_VISCOSITY = 1000;
+	public static final int LAVA_VISCOSITY = 6000;
+	public static final int LAVA_VISCOSITY_NETHER = 2000;
+	/**
+	 * For flowable fluids, the viscosity should match VISCOSITY_RATIO * {@code FlowableFluid.getFlowSpeed}.
```

Comment:
```suggestion
	 * For flowable fluids, the viscosity should match {@code VISCOSITY_RATIO} * {@link net.minecraft.fluid.FlowableFluid#getFlowSpeed()}.
```

Reply:
Can't reference `getFlowSpeed` directly since it's protected.

Reply:
You should be able to reference protected methods from links since they are also public API in Java libraries, so the JD tool should work for them just fine. (I'm 99% sure the JDK itself has a link like that)

Reply:
![image](https://user-images.githubusercontent.com/13494793/160490736-7e9f6ffa-07c3-4f52-a9c9-f572785a7189.png)

Reply:
Going to ignore this warning since it correctly links it in intellij anyway.